### PR TITLE
Add Indonesian translation for 1-introduction

### PR DESCRIPTION
**Pull request by hizzely**
- [x] 1-Introduction
- [x] 1-1 Intro to ML
- [x] 1-2 History of ML
- [x] 1-3 Fairness
- [x] 1-4 Techniques

### CODE REVIEW

**Comment by jlooper** [comment]
hi @hizzely is this ready to merge?

**Comment by hizzely** [comment]
Hi! yes, it's ready. Sorry, I forgot to change the status. Let me change it quick.

**Comment by hizzely** [comment]
One question, do I need to change the assignment link with the translated one?

**Comment by jlooper** [comment]
> One question, do I need to change the assignment link with the translated one?
> Such as `assignment.md` become `assignment.id.md`

hi, yes if you can, please do link to the translated assignments. We'll also want to link to the translated quiz eventually

File: 1-Introduction/translations/README.id.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Di bagian kurikulum ini, Kamu akan berkenalan dengan konsep yang mendasari bidang Machine Learning, apa itu Machine Learning, dan belajar mengenai
+sejarah serta teknik-teknik yang digunakan oleh para peneliti. Ayo jelajahi dunia baru Machine Learning bersama! 
+
+![bola dunia](images/globe.jpg)
+> Foto oleh <a href="https://unsplash.com/@bill_oxford?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Bill Oxford</a> di <a href="https://unsplash.com/s/photos/globe?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+  
+### Pelajaran
+
+1. [Pengantar Machine Learning](1-intro-to-ML/README.md)
```
[the remark this answers is not in the document]
Comment (hizzely) [reply 1-Introduction/translations/README.id.md]: Okay, I've updated those and the assignment links.

**Comment by hizzely** [comment]
Let me know if I should update the link to the images too :)

**Comment by jlooper** [comment]
> Let me know if I should update the link to the images too :)

oh yes, I'm sorry, yes the images need to be relative-path: ../images/image.png for example. thanks!

**Comment by hizzely** [comment]
> 
> 
> > Let me know if I should update the link to the images too :)
> 
> oh yes, I'm sorry, yes the images need to be relative-path: ../images/image.png for example. thanks!

Okay, it's done! Let me know again if I need to change something else

**Review by jlooper — APPROVED** [review]
🙏🚀

**Pull request merged**